docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+
+# Install collectd and dependencies for fritzcollectd.
+RUN apk add --no-cache \
+        collectd \
+        collectd-python \
+        collectd-network \
+        python2 \
+        py2-pip \
+        py2-lxml \
+        py2-pbr \
+        git
+
+# Copy entrypoint script.
+COPY entrypoint.sh /
```

Comment:
``/usr/local/bin/`` is a way cooler place cause it is in path. ;) Also hadolint suggests to use array notation like ``COPY [ "source", ... , "dest" ]